dlp/src/test/java/dlp/snippets/InspectTests.java

```diff
@@ -251,6 +251,42 @@ public void testInspectStringCustomHotwordNegativeExample() throws Exception {
     assertThat(output, not(containsString("John Doe")));
   }
 
+  @Test
+  public void testInspectStringMultipleRulesPatientRule() throws Exception {
+    InspectStringMultipleRules.inspectStringMultipleRules(PROJECT_ID,
+        "patient: Jane Doe");
+
+    String output = bout.toString();
+    assertThat(output, containsString("VERY_LIKELY"));
+  }
+
+  @Test
+  public void testInspectStringMultipleRulesDoctorRule() throws Exception {
+    InspectStringMultipleRules.inspectStringMultipleRules(PROJECT_ID,
+        "doctor: Jane Doe");
+
+    String output = bout.toString();
+    assertThat(output, containsString("Findings: 0"));
```

Comment:
Shouldn't there be a finding here?

Reply:
No, because the custom rule sets it as UNLIKELY which is below the default threshold for results to return.

Reply:
What are the other potential likelihoods? Unlikely to me sounds like "probably not but possibly," not "ignore anything found".

Reply:
https://cloud.google.com/dlp/docs/likelihood

The minimum likelihood to return is configurable (minLikelihood property on InspectConfig) and defaults to POSSIBLE.

Reply:
Oh I see, so does "setFixedLikelihood" mean that all results that match the rule will have that as a likelihood, instead of whatever likelihood it would have assigned?

Reply:
Correct. It's adding a rule that says "when a match has this hotword nearby, set the likelihood to X".